selfdrive/ui/qt/maps/map.cc

```diff
@@ -229,7 +218,7 @@ void MapWindow::updateState(const UIState &s) {
     }
   }
 
-  if (sm.rcv_frame("navRoute") != route_rcv_frame) {
+  if (sm.updated("navRoute")) {
```

Comment:
I forgot the exact reason, but the two code blocks for navRoute are currently required (something to do with the map not being ready, and this calls map-specific stuff). Is this fixed?